### PR TITLE
Update database config comments

### PR DESCRIPTION
**Pull request by svbergerem**

### CODE REVIEW

**Review by Flaburgan — APPROVED** [review]
Nice check thank you!

**Review by SuperTux88 — APPROVED** [review]

File: config/database.yml.example [hunk 1]
```diff
@@ -17,8 +17,8 @@ mysql: &mysql
   collation: utf8mb4_bin
 
 
-# Comment the the mysql line and uncomment the postgres line
-# if you want to use postgres
+# Comment the the postgresql line and uncomment the mysql line
```
Comment (SuperTux88) [inline config/database.yml.example]: Oh, can you also remove the double "the the"?

**Review by SuperTux88 — CHANGES_REQUESTED** [review]

File: config/database.yml.example (hunk 1 above)
Comment (svbergerem) [inline config/database.yml.example]: done

**Review by SuperTux88 — APPROVED** [review]

**Comment by SuperTux88** [comment]
Merged as b508b9f6ef7282e782e91663705637bc3af87a94

Thanks 👍

**Pull request closed without merging**